tests/test_pipeline_controller.py

```diff
@@ -10,9 +10,9 @@
 from tests.test_helpers import TestHelpers
 
 
-_CREATION_DT = '2021-01-{} 01:01:01-05'
-_UPDATE_DT = '2021-02-{} 02:02:02-05'
-_DELETION_DATE = '2021-03-{}'
+_CREATION_DT = '2021-01-0{}T01:01:01-05:00'
+_UPDATE_DT = '2021-02-0{}T02:02:02-05:00'
```

Comment:
So all of this still pass through a DST change because you're using a custom tz? (i.e. when "ET" resolves to a `-04:00` offset in Nov?)

Reply:
Yeah I believe so -- in order to match the PostgreSQL behavior, I'm using EST which always resolves to `-05:00`. I have vague plans to do a total overhaul of the dates and store everything in UTC but for now this is the standard.